client/me/purchases/manage-purchase/payment-method-selector/index.tsx

```diff
@@ -53,7 +60,7 @@ function useLogError( message: string ): CheckoutPageErrorCallback {
 				extra: {
 					env: config( 'env_id' ),
 					type: 'payment_method_selector',
-					message: String( errorMessage ),
+					message: convertErrorToString( errorMessage ),
```

Comment:
Should we rename the `errorMessage` argument? Feels like an unexpected name for an error object.